packages/rest-typings/src/v1/users/UsersUpdateParamsPOST.ts

```diff
@@ -16,7 +16,6 @@ export type UsersUpdateParamsPOST = {
 		nickname?: string;
 		statusText?: string;
 		roles?: string[];
-		joinDefaultChannels?: boolean;
```

Comment:
We can't just remove the parameter right now in a sudden manner, it's considered a breaking change. You can still keep the client-side changes though.

Reply:
You are right, I'll revert this.